src/argus/notificationprofile/media/email.py

```diff
@@ -140,19 +140,15 @@ def has_duplicate(self, queryset: QuerySet, settings: dict) -> bool:
         """
         return queryset.filter(settings__email_address=settings["email_address"]).exists()
 
-    @staticmethod
-    def send(event: Event, destinations: QuerySet[DestinationConfig], **_) -> bool:
-        """
-        Sends email about a given event to the given email destinations
-
-        Returns False if no email destinations were given and
-        True if emails were sent
-        """
-        email_destinations = destinations.filter(media_id=EmailNotification.MEDIA_SLUG)
-        if not email_destinations:
-            return False
+    @classmethod
+    def get_relevant_addresses(cls, destinations: List(DestinationConfig)) -> List:
```

Comment:
Returns a List of what? Also:

```suggestion
    def get_relevant_addresses(cls, destinations: List[DestinationConfig]) -> List:
```